src/main/resources/cc/r/runAnalysis.R

```diff
@@ -0,0 +1,39 @@
+library(devtools)
+
```

Comment:
Where are all the changes which were applied to PLE / PLP R scripts? (`--no-mutiarch`, install w/o devtools, drivers for BQ / Impala, etc)?